Website/plugins/git-reflog/add-ref-hash.raku

```diff
@@ -4,10 +4,16 @@ sub ($source-cache, $mode-cache, Bool $full-render,  $source-root, $mode-root, %
     my $rep = $source-root.IO.parent;
     my $proc = run <<git -C $rep rev-parse --short HEAD>>, :out;
     my $reflog = $proc.out.slurp(:close).chomp;
+    if $reflog {
```

Comment:
Can you add a comment here explaining why/how this works

Reply:
here I am just creating a soft fail patch. If $reflog has not been giving a value by the 'git' call, then the html defaults to 'raku/docs/doc/` as suggested by @coke. 

As to someone who has just a tar archive, the build process clones the `Raku/docs` repo. Otherwise no html at all would be generated. 

The git reflog is to show which version of the git repo is being built.